apps/mula/lib/components.ex

```diff
@@ -78,4 +78,78 @@ defmodule Mula.Components do
     </div>
     """
   end
+
+  attr(:value, :float,
+    required: true,
+    doc:
+      "the value represented on the meter, must be between min_value and max_value, if min_value and max_value are not passed, value will be a plain percentage"
+  )
+
+  attr(:max_value, :float,
+    default: 100.0,
+    doc:
+      "the maximun value of the interval represented on the meter, must be greater than min_value, must be greater than or equal value"
+  )
+
+  attr(:min_value, :float,
+    default: 0.0,
+    doc:
+      "the minimun value of the interval represented on the meter, must be lesser than max_value, must be lesser than or equal value"
+  )
+
+  attr(:value_text_formatter, :any,
+    default: nil,
+    doc:
+      "a function that receives the calculated percentage and returns a string, it's used to set the aria-valuetext and the value_text returned"
+  )
+
+  slot(:inner_block, doc: "the optional inner block that renders the")
+
+  def meter(assigns) do
+    %{
+      value: value,
+      max_value: max_value,
+      min_value: min_value,
+      value_text_formatter: value_text_formatter
+    } = assigns
```

Comment:
I have a feeling that just doing assign.value is better to read than extracting like that.